veg2hab/vegkartering.py

```diff
@@ -1175,7 +1208,15 @@ def from_editable_vegtypes(cls, gdf: gpd.GeoDataFrame) -> Self:
                 *gdf.columns[gdf.columns.str.startswith(("SBB", "VvN", "perc"))],
             ]
         )
-        return cls(gdf)
+
+        gdf["_state"] = gdf["_state"].apply(KarteringState)
+
+        kartering = cls(gdf)
+        assert kartering.has_state(
```

Comment:
Joh, iets overkill deze assert. Maar is prima.

Reply:
Zeg je expliciete state checking dan krijg je het ook ;)